src/main/java/uk/gov/justice/hmpps/datacompliance/web/dto/ManualRetentionReasonDisplayName.java

```diff
@@ -27,6 +27,17 @@
     private ManualRetentionReasonCode reasonCode;
 
     @ApiModelProperty(value = "The UI display name")
+    @NotNull
```

Comment:
Just mentioning in case there are other DTO's as well as this one.  No examples or position in swagger docs/annotations, meh.

Reply:
I'll look to flesh it out, good point